handler/handler.ts

```diff
@@ -1,28 +1,47 @@
 import { Request, Response } from 'express';
-import { getUsers } from '../controller/controller';
-import jwt from 'jsonwebtoken'; // Importar el módulo jsonwebtoken
+import jwt, { JwtPayload } from 'jsonwebtoken';
+import { UsersController } from '../controller/controller';
+import { TransformedUser } from '../models/models';
+import { JWT_MID } from '../middleware/middleware';
 
 const SECRET_KEY = 'gauss626';
 
-const getUsersHandler = async (req: Request, res: Response) => {
+async function getUsersHandler(): Promise<TransformedUser[]> {
   try {
-    const transformedUsers = await getUsers();
-    res.json(transformedUsers);
+    const usersController = new UsersController();
+    const users = await usersController.getUsers();
+    return users;
   } catch (error) {
-    console.error('Error al llamar a la API:', error);
-    res.sendStatus(500); // Internal Server Error
+    throw new Error('Error al obtener los usuarios');
   }
-};
+}
 
-const loginHandler = (req: Request, res: Response) => {
-  // Generar y devolver el token JWT en la respuesta
-  const payload = {
-    username: 'example',
-    role: 'admin'
+export class Handler {
```

Comment:
Why do you have a Handler class and then a `getUsersHandler()` above? Which one is the one you're going to use?

Reply:
I will use Handler